projects/fsastorefrontlib/src/core/change-request/facade/change-request.service.ts

```diff
@@ -66,19 +64,17 @@ export class ChangeRequestService {
     );
   }
 
-  loadChangeRequest() {
+  loadChangeRequest(requestId: string) {
     this.userIdService
       .getUserId()
       .pipe(take(1))
       .subscribe(occUserId => {
-        if (this.requestId) {
```

Comment:
This global variable `requestId ` causes every time we dispatch an action with the same requestId, instead of the new one.